docs/en/connector-v2/sink/Hive.md

```diff
@@ -33,7 +33,7 @@ By default, we use 2PC commit to ensure `exactly-once`
 |             name              |  type   | required | default value  |
 |-------------------------------|---------|----------|----------------|
 | table_name                    | string  | yes      | -              |
-| metastore_uri                 | string  | yes      | -              |
+| hive_jdbc_url                 | string  | yes      | -              |
```

Comment:
How to be compatible with older versions?

Reply:
it not compatible with old version, because i don't want both use hive2 jdbc and hive metastore. so i removed the metastore only use jdbc.

Reply:
> it not compatible with old version, because i don't want both use hive2 jdbc and hive metastore. so i removed the metastore only use jdbc.

As an open source project we have to consider the compatibility of features, we know that many users are using Hive Connector, in order to be compatible with those older users, I think it is a better way to support both jdbc and metastore.

Reply:
yes.
but if we want use savemode feature only with hive metastore, is difficulty to create table. 
like table format, bucket setting, table location etc. we need lots of parameter to config them.
so i want use sql template to let user define the template we can replace table name, columns in this template and run the sql to create table.

I can add metastore_url back, and it will make the code more easier. But user need both config `jdbc` and `thrift` on hive connector.

Reply:
> yes. but if we want use savemode feature only with hive metastore, is difficulty to create table. like table format, bucket setting, table location etc. we need lots of parameter to config them. so i want use sql template to let user define the template we can replace table name, columns in this template and run the sql to create table.
> 
> I can add metastore_url back, and it will make the code more easier. But user need both config `jdbc` and `thrift` on hive connector.

Yes, you can add `metastore_url ` tell users `save mode` only can be use when `jdbc_url` is configed.

Reply:
done. now if pass hive_jdbc_url. will create catalog and run savemode function. 
if not pass this parameter, will ignore.